tests/e2e-openshift-ossm/ossm-monolithic-otel/install-otel-collector.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: opentelemetry.io/v1alpha1
```

Comment:
What is the purpose of this OTEL Collector? I don't see where the test sends traces to this collector

Reply:
Currently Tempo Monolithic doesn't support Zipkin receiver. Thus we have to use a OTEL collector to receive traces using a Zipkin receiver and forward it to the Monolithic instance. The OTEL collector is specified in the OSSM CR. https://github.com/grafana/tempo-operator/pull/866/files#diff-f7c021630e6a1eb8ae1c18159b0000e6d5e4de555630e800ab0e6f0799a982d7R33